enlighten/measurement/Measurements.py

```diff
@@ -385,19 +385,34 @@ def export_session(self, filename=None, prompt=True):
         # currently, all Sessions are stored in ~/EnlightenSpectra
         directory = common.get_default_data_dir()
 
-        # doesn't use the dict
-        if self.ctl.save_options.save_csv():
-            self.export_session_csv(directory, filename, visible_only=visible_only)
+        # warn user if they are about to overwrite an existing file
+        # it looks like only risk is for .csv and .json extensions
+        file_exists = (os.path.exists(os.path.join(directory, f'{filename}.csv'))
+                       or os.path.exists(os.path.join(directory, f'{filename}.json')))
```

Comment:
Technically, I suppose .spc could go in there too. I don't think many people export to .spc but _shrug._

Reply:
Got it.  Will change. I had all the export boxes checked, but I did't have the .spc (or jcamp) writer installed correctly so it wasn't exporting.